AdobeStockImageAdminUi/etc/module.xml

```diff
@@ -6,5 +6,9 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_AdobeStockImageAdminUi" />
+    <module name="Magento_AdobeStockImageAdminUi">
```

Comment:
I don't think there is need to add `Magento_Ui` module to sequence, there is no hard installation or configuration dependency. Soft dependency should be enough (composer.jsoon)

Reply:
@sivaschenko, I have fixed it. Thanks!

Reply:
Thank you!